resources/js/components/Publish/PublishForm.vue

```diff
@@ -158,8 +158,8 @@
                     class="rtl:mr-4 ltr:ml-4 btn-primary flex items-center"
                     :disabled="!canPublish"
                     @click="confirmingPublish = true">
-                    <span v-text="__('Publish')" />
-                    <svg-icon name="micro/chevron-down-xs" class="rtl:mr-2 ltr:ml-2 w-2" />
+                    <span v-text="this.canManagePublishState ? __('Publish') : __('Create Revision')" />
```

Comment:
Like we've done in Core, can we extract this out into a `publishButtonText` method?